packages/hardhat-plugin-ethers/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
```

Comment:
We need to consider whether this should just be populated with the existing hardhat-plugin changelog and similarly for the viem package.